core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -186,6 +189,7 @@ public Server createServer() throws ServletException {
     combinedMetricsTags.putAll(configuredTags);
 
     configureBaseApplication(resourceConfig, combinedMetricsTags);
+    configureResourceExtensions(resourceConfig);
     setupResources(resourceConfig, getConfiguration());
```

Comment:
an application already has to implement setupResources to add resources, maybe we should let the application decide whether to configure resource extensions before or after it adds its own resources?

Reply:
As per the last PR this is about allowing users of some product to extend it using config, rather than some user of rest-utils.